nilrt_snac/_Configs/_ConfigFile.py

```diff
@@ -0,0 +1,65 @@
+"""Helper class to read/write and update configuration files."""
```

Comment:
It's a little confusing that this utility class is in the `_Configs` module when it is not a config. Can it be moved to into the main package?

Reply:
I like to scope classes and objects at the lowest required level. Only Configs use this class which is why I have it in the _Configs folder. I can move it if you have a strong preference.

Reply:
Nah. I suppose that's fine. Dropped.